styles.css

```diff
@@ -2,7 +2,8 @@
     --primary-color:black;
     --logo-color:#f2f2f2;
     --white-color:#fff;
-    --icon-badge:#0000ee;
+    --icon-badge:#3b82f6;
```

Comment:
You can use css variables

Reply:
these are css variable declarations only.